Code/2022/data_2022.Rmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "PISA 2022 Data Import"
+author: "Guan Ru Chen and Shabarish"
+date: "12/8/2024"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r packages}
+library(here)
+library(tidyverse)
+library(haven)
+```
+
+Use the Haven package to read in the SPSS files.
+
+```{r read in spss sav files}
+urls <- read_csv(here("Code/download_urls.csv")) |> 
```

Comment:
This file was not updated as part of this PR.

Reply:
download_urls file is now uploaded, we'll work on the other files afterward